### PR TITLE
Fix test assert and temporarily disable

### PR DESCRIPTION
**Pull request by joakim-hove**
Part of: https://github.com/OPM/opm-common/pull/2347

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: tests/parser/RestartConfigTests.cpp [hunk 1]
```diff
@@ -314,7 +314,11 @@ RESTART=0
     auto sched = make_schedule(deckData1, false);
 
     BOOST_CHECK(  sched.write_rst_file( 0 ) );
-    BOOST_CHECK( !sched.write_rst_file( 1 ) );
+    // By comparision with Eclipse it turns out that we should write a restart
+    // file for report step 0, there is clearly a bug in the code old RPTRST
```
Comment (bska) [inline tests/parser/RestartConfigTests.cpp:318]: Did you mean "report step 1" here?

Comment (joakim-hove) [reply tests/parser/RestartConfigTests.cpp:318]: Yes

Comment (joakim-hove) [reply tests/parser/RestartConfigTests.cpp]: Yes!

**Comment by joakim-hove** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Looks good to me.  I'll merge into master.

**Pull request merged**